platform/darwin/src/MGLAccountManager.m

```diff
@@ -28,6 +30,11 @@ + (void)load {
     if (accessToken.length) {
         self.accessToken = accessToken;
     }
+    
+    NSString *apiBaseURL = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLMapboxAPIBaseURL"];
+    if (apiBaseURL.length) {
```

Comment:
I wonder if we should be checking to see if `apiBaseURL` is a valid(ish) URL, rather than a length check. Not sure if that logic already exists elsewhere in core, though.